doc/rtd/topics/datasources/ec2.rst

```diff
@@ -80,6 +80,9 @@ The settings that may be configured are:
  * **timeout**: the timeout value provided to urlopen for each individual http
    request.  This is used both when selecting a metadata_url and when crawling
    the metadata service. (default: 50)
+ * **configure_secondary_ips**: Boolean (default: True) to allow cloud-init
+   to configure any secondary IPs described by the metadata service.
+   On Ubuntu Xenial, Bionic and Eoan the default is False.
```

Comment:
Let's drop the release names from the docs; they will get stale and there is no mention of other distros.

Reply:
+1 or above say enabled on Focal and newer

Reply:
let's drop as this might change too once we decide whether it's worth SRUing this to Bionic Xenial etc .